### ISSUE TITLE
Pydantic Validation error in get_accounts

### ISSUE THREAD

**Issue by hecrogon** [klaviyo/klaviyo-api-python#68]
Hey guys,
I noticed that if the organization has an empty value in the Street Address 2 field, the `accounts_api.get_accounts` method raises a Validation error. Would be possible to change the specifications and make that StrictStr field (and maybe others) also optional? 

```
   File "/usr/local/lib/python3.12/site-packages/openapi_client/api/accounts_api.py", line 452, in get_accounts
    return self.api_client.response_deserialize(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/openapi_client/api_client.py", line 325, in response_deserialize
    return_data = self.deserialize(response_text, response_type)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/openapi_client/api_client.py", line 417, in deserialize
    return self.__deserialize(data, response_type)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/openapi_client/api_client.py", line 462, in __deserialize
    return self.__deserialize_model(data, klass)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/openapi_client/api_client.py", line 790, in __deserialize_model
    return klass.from_dict(data)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/openapi_client/models/get_account_response_collection.py", line 97, in from_dict
    "data": [AccountResponseObjectResource.from_dict(_item) for _item in obj["data"]] if obj.get("data") is not None else None,
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/openapi_client/models/account_response_object_resource.py", line 98, in from_dict
    "attributes": AccountResponseObjectResourceAttributes.from_dict(obj["attributes"]) if obj.get("attributes") is not None else None,
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/openapi_client/models/account_response_object_resource_attributes.py", line 120, in from_dict
    "contact_information": ContactInformation.from_dict(obj["contact_information"]) if obj.get("contact_information") is not None else None,
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/openapi_client/models/contact_information.py", line 96, in from_dict
    "street_address": StreetAddress.from_dict(obj["street_address"]) if obj.get("street_address") is not None else None
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/openapi_client/models/street_address.py", line 88, in from_dict
    _obj = cls.model_validate({
           ^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/pydantic/main.py", line 593, in model_validate
    return cls.__pydantic_validator__.validate_python(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
pydantic_core._pydantic_core.ValidationError: 1 validation error for StreetAddress
address2
  Input should be a valid string [type=string_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.9/v/string_type
```

**Comment by ben-horgan-klaviyo** [comment]
Thanks for bringing this up. We'll work on a fix. In the mean time, you can use our [dictionary response option](https://github.com/klaviyo/klaviyo-api-python/blob/main/CHANGELOG.md#backwards-compatibility) to work around this error.

**Comment by hecrogon** [comment]
Awesome. Thank you so much for the fast answer

**Comment by ben-horgan-klaviyo** [comment]
Fixed in v13.0.0
